app/models/user.rb

```diff
@@ -249,6 +249,10 @@ def full_name
     "#{self.first_name} #{self.last_name}"
   end
 
+  def username
```

Comment:
Yes, we understand this issue, it's a tecnopolitical decision at the moment. 
It's not so problematic because: 
a) on this phase we'll only give access to trusted users 
b) on Participa we save every change that the user make (paper_trail) so impersonation could be detected.